src/SDKs/Batch/Management/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag />
```

Comment:
@xingwu1 can you build your management sdk as below, so that the meta data is updated
msbuild build.proj /t:Build /p:Scope=SDKs\Batch\Management

Reply:
@shahabhijeet this props is just generated with this PR. If I run it now, there is no change.